Add `osu!(lazer) Updates: December 9, 2024` news post (#12541)
ppy/osu-wiki: news/2024/2024-12-09-osulazer-updates-december-9-2024.md

---
layout: post
title: "osu!(lazer) Updates: December 9, 2024"
date: 2024-12-09 15:00:00 +0000
---

> Cure your post-world cup depression with some brand new osu!(lazer) updates!

Reviewer: ```suggestion
Cure your post–world cup depression with some brand-new osu!(lazer) updates!
```

![](/wiki/shared/news/2024-12-09-osulazer-updates-december-9-2024/banner.jpg)

> While the community has been indulging themselves to some top of the line osu! gameplay for the past few weeks, the developers stayed hard at work, making osu!(lazer) even better.

Reviewer: ```suggestion
While top players have been hard at work making world cup history for the past few weeks, the developers also stayed hard at work, making osu!(lazer) even better.
```

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/E7cslGdXKLk" frameborder="0" allowfullscreen></iframe>
</div>

### Editor changes

- **The visual appearance of highlighted osu!mania notes has been improved.** Cleans up some sub-par bounding boxes for objects, and makes selections easier to see.

> - **osu!mania notes won't disappear when reaching their hit time.** Another small annoyance that didn't make sense in the editor.

Reviewer: ```suggestion
- **osu!mania notes won't disappear when scrolled to anymore.** Another small annoyance that didn't make sense in the editor.
```

- **Some default settings for beatmaps have changed.** Namely, `Enable countdown` and `Samples match playback rate` options are disabled by default, matching osu!(stable).
- **Objects will stay selected if you control-click on empty space.** If your aim is lacking, we have you covered (in the editor).

### Gameplay changes

- **The idle time before gameplay loads has been increased when hovering controls.** Now there's more time to ponder whether the background dim percentage is just right.
- **A toggle has been added for star fountains during gameplay.** No more star fountain jump scares.

![](/wiki/shared/news/2024-12-09-osulazer-updates-december-9-2024/star-fountain.jpg)

> - **Your score will be saved even if you quick-restart after passing a map.** R.I.P. to all the sub-par runs that could've been saved so far, at least until they've been superseded.

Reviewer: ```suggestion
- **Your score will be saved even if you quick-restart after passing a map.** Rest in peace to all the subpar runs that were tossed into the void so far.
```

- **The "watch replay" button now works on first press.** No, your mouse wasn't broken after all.

> - **A new osu! gameplay mod has been implemented, "Bloom".** They say everything's bigger in... osu!(lazer)?

Reviewer: ```suggestion
- **New "Bloom" mod has been added.** They say everything's bigger in... osu!(lazer)?
```

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2024-12-09-mod-bloom.mp4" type="video/mp4" preload="none">
    </video>
</div>

> - **Stage line alignment in osu!(lazer) now matches osu!(stable).** It's small things like these that can make all the difference for players.

Reviewer: idk if pixel-level adjustments are worth noting <.<

```suggestion
```

Author: fair

- **You can now get an SS in osu!mania even if there are GREAT judgements.** It's big things like these that *do* make all the difference for players.

> - **The scroll bar for adjusting osu!mania scroll speed is now more granular.** The speed can now be adjusted with single decimal precision.

Reviewer: ```suggestion
- **The scroll bar for adjusting osu!mania scroll speed is now more granular.** The speed can now be adjusted with single-decimal precision.
```

- **osu!taiko's constant scroll algorithm has been adjusted.** The speed more closely matches player expectations.

### Other changes

> - **You can now install osu!(lazer) directly from osu!(stable)'s cutting edge release.** While not necessarily an osu!(lazer) change, it shows that slowly, but surely, osu!(lazer) is getting into a state ready for major transition.

Reviewer: mm long

```suggestion
- **You can now install osu!(lazer) directly from osu!(stable)'s cutting edge release.** While maybe still not the biggest nudge for people to transition, we're getting there.
```

![](/wiki/shared/news/2024-12-09-osulazer-updates-december-9-2024/try-lazer.jpg)

- **The public chats are now sorted alphabetically, and private chats are sorted based on recent messages.** Order has been... added?
- **Your rank is now accurately presented based on the mode you are playing.** Previously only the rank of your default game mode would be visible in-game.

> - **Playlists can now be closed within a grace period after creation.** If you spot a mistake or change your mind on a playlist within the first 5 minutes, the playlist can be closed with the new `Close` button.

Reviewer: ```suggestion
- **Playlists can now be closed within 5 minutes after creation.** A more sane alternative than having to poke peppy.
```

- **Hold-for-pause during gameplay has been re-added.** This time it's a toggle, meaning that sometimes you *can* get the best of both worlds.

![](/wiki/shared/news/2024-12-09-osulazer-updates-december-9-2024/hold-to-pause.jpg)

- **A map's maximum achievable pp can now be displayed with a skin element.** Select the `MaxPP` attribute for the `BeatmapAttributeText` skin element in the skin editor, and voilà.

> - **Player settings have been added to the spectator and replay screens.** This means that you can now adjust the same settings as if you were the one playing.

Reviewer: replays always had these settings

```suggestion
- **Gameplay settings have been added to the spectator screens.** This is the same menu that appears before playing a map or while watching a replay.
```
